src/python/CMakeLists.txt

```diff
@@ -51,6 +55,17 @@ if(BUILD_WHEEL)
     ${WHEEL_TARGET_NAME}
     COMMENT "Copying files to wheel directory: ${WHEEL_TARGET_NAME}"
   )
+  add_custom_command(TARGET python POST_BUILD
+    # Use a new directory to fix the onnxruntime_genai runtime dependency and avoid contamination
+    COMMAND ${CMAKE_COMMAND} -E copy_directory ${WHEEL_TARGET_NAME} ${GENERATE_TYPE_DIR}
+    COMMAND ${CMAKE_COMMAND} -E copy_directory ${ORT_LIB_DIR} ${GENERATE_TYPE_DIR}
+    # Be compatible with both system-wide but unprivileged env and virtual env where --user is not available
+    COMMAND ${PYTHON_EXECUTABLE} -m pip install --user pybind11-stubgen numpy || ${PYTHON_EXECUTABLE} -m pip install pybind11-stubgen numpy
```

Comment:
Is it better to call pip install in cmake for generating the stub, or would it make sense to simply maintain a pyi file manually? I feel that the process is over-complicated given there is a fix-up step needed in src/python/fix_pyi.py